cmd/lncli/main.go

```diff
@@ -3,11 +3,27 @@ package main
 import (
 	"fmt"
 	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/roasbeef/btcutil"
 	"github.com/urfave/cli"
 
+	flags "github.com/btcsuite/go-flags"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	defaultConfigFilename  = "lnd.conf"
+	defaultSSLCertFilename = "ssl.cert"
+)
+
+var (
+	lndHomeDir        = btcutil.AppDataDir("lnd", false)
```

Comment:
Yeah it can't be used as is currently. However, when we restructure the packages within `lnd` to accommodate your `gomobile` work @halseth we should be able to re-use the definitions.